### PR TITLE
Modify class of Revit_Core_Engine covered with descriptions

### PR DESCRIPTION
**Pull request by pawelbaran**
### Issues addressed by this PR

3rd part of #596

### Test files

This PR is only about descriptions.

### Additional comments

In the coming days, this PR will be followed with sister PRs covering other classes of Revit_Core_Engine.

### CODE REVIEW

**Comment by pawelbaran** [comment]
@BHoMBot check compliance

**Comment by pawelbaran** [comment]
@BHoMBot check compliance

File: Revit_Core_Engine/Modify/SetType.cs [hunk 1]
```diff
@@ -132,7 +154,7 @@ public static bool ISetType(this Element element, IBHoMObject bHoMObject, RevitS
         /****              Private Methods              ****/
         /***************************************************/
 
-        public static bool TrySetTypeFromString(this Element element, IBHoMObject bHoMObject, RevitSettings settings)
+        private static bool TrySetTypeFromString(this Element element, IBHoMObject bHoMObject, RevitSettings settings)
```
Comment (tiagogrossi) [inline Revit_Core_Engine/Modify/SetType.cs:157]: Can I suggest you add a description in this private class as well? Just for due diligence. Also check a couple of compliance warnings.

Comment (pawelbaran) [reply Revit_Core_Engine/Modify/SetType.cs:164]: Fair suggestion @tiagogrossi. Descriptions added!

**Comment by pawelbaran** [comment]
@BHoMBot check required

**Comment by pawelbaran** [comment]
@BHoMBot check copyright-compliance
@BHoMBot check dataset-compliance
@BHoMBot check ready-to-merge

**Review by tiagogrossi — APPROVED** [review]
LGTM

**Comment by tiagogrossi** [comment]
@BHoMBot check ready-to-merge

**Pull request merged**